apps/mobile/src/state/marketplace/atom.ts

```diff
@@ -101,9 +100,8 @@ export const myOffersAtom = focusAtom(offersAtom, (optic) =>
 export const myOffersSortedAtom = atom((get) => {
   const sortingOptions = get(selectedMyOffersSortingOptionAtom)
   const myOffers = get(myOffersAtom)
-  const idsOfRequestedOffers = get(idsOfRequestedOffersAtom)
 
-  return sortOffers(myOffers, sortingOptions, idsOfRequestedOffers)
+  return sortOffers(myOffers, sortingOptions)
```

Comment:
Sorting by requests is dropped. Offers are always ordered by the selected filters, user's interaction should not have any impact on how the offers in marketplace are sorted